### ISSUE TITLE
How do I customise output?

### ISSUE THREAD

**Issue by lllama** [OpenGrok/OpenGrok#952]
I can't seem to find any docs or discussions about how to customise the various pages in OpenGrok. Where should I be looking? I'm particularly interested in changing the page headers and the xref pages.

Thanks in advance.

**Comment by vladak** [comment]
Section 5.4.2 in the README file describes how to insert into header and footer.

What exactly do you need to change in the xref pages ?

**Comment by vladak** [comment]
Another thing which can be changed is the body of the index page, simply change `index_body.html` under the webapp directory.

**Comment by vladak** [comment]
Yes another thing which can be changed is the CSS style, one of the `default`, `offwhite` and `polished` however I don't know how to change them at the moment.

**Comment by lllama** [comment]
Thanks @vladak. I'm experimenting with adding in some additional javascript, so putting that into the header or footer should be sufficient. I was also planning on tweaking the markup of various pages, hence wondering about where they were generated.

**Comment by lllama** [comment]
I looks like some of what I want is in src/org/opensolaris/opengrok/web/Util.java (readableLine in particular). I shall investigate further.

**Comment by vladak** [comment]
Any results from the investigation ?

Anyway, I will keep this issue open to track the documentation of the CSS styles.
